### PR TITLE
values: change default domain

### PR DESCRIPTION
**Pull request by m-ildefons**
- Change default domain to the stadard `example.com`, instead of the more esoteric `.howdoi.website`
- Break up long lines

- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [x] CHANGELOG.md has been updated should there be relevant changes in this PR.

### CODE REVIEW

File: charts/s3gw/values.yaml [hunk 1]
```diff
@@ -45,7 +47,7 @@ ui:
   # 'serviceName' is the service name of the S3GW user interface.
   serviceName: "s3gw-ui"
   # 'publicDomain' is the public domain of the UI Service used by the Ingress.
-  publicDomain: "fe.127.0.0.1.omg.howdoi.website"
+  publicDomain: "example.com"
```
Comment (giubacc) [inline charts/s3gw/values.yaml:50]: I'd rather prefer the user to be forced to put an explicit value.
I think it is better, for the public domain variables, to require the user to make the choice.

We can implement the following logic inside a `chart-validations.yaml` file:

```
{{- if (empty .Values.ui.publicDomain) }}
{{- fail "ui.publicDomain cannot be empty" }}
{{- end }}
```

File: charts/s3gw/values.yaml [hunk 2]
```diff
@@ -61,15 +63,20 @@ useExistingSecret: false
...
-# Set this as the empty string to make the Chart to compute a random alphanumeric value.
+# 'secretKey' is the S3 Secret Key; the value is used when
+# `useExistingSecret: false`.
+# Set this as the empty string to make the Chart to compute a random
+# alphanumeric value.
 secretKey: "test"
 # 'publicDomain' is the public domain of S3GW used by the Ingress
-publicDomain: "be.127.0.0.1.omg.howdoi.website"
-# 'privateDomain' is the private domain of S3GW used inside the Kubernetes cluster
+publicDomain: "example.com"
```
Comment (giubacc) [inline charts/s3gw/values.yaml:77]: I'd rather prefer the user to be forced to put an explicit value.
I think it is better, for the public domain variables, to require the user to make the choice.

We can implement the following logic inside a `chart-validations.yaml` file:

```
{{- if (empty .Values.publicDomain) }}
{{- fail "publicDomain cannot be empty" }}
{{- end }}
```

**Review by giubacc — CHANGES_REQUESTED** [review]

File: charts/s3gw/values.yaml (hunk 1 above)
Comment (m-ildefons) [reply charts/s3gw/values.yaml:50]: Sure, sounds like a good idea.
Can you point me to the docs for the `chart-validation.yaml` file? I only found https://helm.sh/docs/topics/charts/#schema-files schema files, which don't seem to be the same thing.

Comment (giubacc) [reply charts/s3gw/values.yaml:50]: we did a very similar thing in Epinio, looks here: https://github.com/epinio/helm-charts/blob/main/chart/epinio/templates/chart-validations.yaml

**Review by giubacc — APPROVED** [review]
LGTM

**Pull request merged**